src/app/tests/suites/certification/Test_TC_OTCCM_2_1.yaml

```diff
@@ -0,0 +1,230 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 3.3 [TC-OTCCM-2.1] Change to Mode functionality with DUT as Server
+
+PICS:
+    - OTCCM.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Oven Mode"
+    endpoint: 1
+
+    #PIXIT.OVEN.MODE_CHANGE_OK
+    NewMode_CHANGE_OK:
+        type: int8u
+        defaultValue: 2
+    #PIXIT.OVEN.MODE_CHANGE_FAIL
+    NewMode_CHANGE_FAIL:
+        type: int8u
+        defaultValue: 10
+    invalid_mode_th:
+        type: int8u
+        defaultValue: 5
+
+tests:
+    - label:
+          "Step 1:Commission DUT to TH (can be skipped if done in a preceding
+          test)."
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Step 2: TH reads from the DUT the SupportedModes attribute."
+      PICS: OTCCM.S.A0000
+      command: "readAttribute"
+      attribute: "SupportedModes"
+      response:
+          saveAs: "suppoted_mode_dut"
```

Comment:
there's a spelling error in the variable name here.  Is this used anywhere?

Reply:
nope, not used, but still I corrected it